docs/vectorizer.md

```diff
@@ -144,9 +149,12 @@ SELECT
    chunk,
    embedding <=> <query embedding> as distance
```

Comment:
```suggestion
   embedding <=> ai.openai_embed(...) as distance
```

Since we are now adding secret fetching to cloud we can do the openai embedding call here. This is much nicer because it allows copy pasting the docs and actually doing semantic search.

Can be a separate PR though to change this.